airflow/jobs/scheduler_job.py

```diff
@@ -699,6 +699,11 @@ def _run_scheduler_loop(self) -> None:
             self.adopt_or_reset_orphaned_tasks,
         )
 
+        timers.call_regular_interval(
+            conf.getfloat('scheduler', 'trigger_timeout_check_interval', fallback=15.0),
```

Comment:
```suggestion
            conf.getfloat('scheduler', 'trigger_timeout_check_interval'),
```

This default should be moved into `airflow.config_templates/config.yml` so it gets properly documented everywhere. You then won't need the fallback here because it'll use the value from `default_airflow.cfg`.

Reply:
I moved it into the config YAML, but kept the fallback here so it matches the other conf calls above it.